src/components/menu/Menu.css

```diff
@@ -1,17 +1,21 @@
 .menu {
-  top: 2rem;
-  left: 2rem;
   display: flex;
-  height: 5vh;
-  border-bottom: 1px solid white;
-  box-sizing: border-box;
+  padding: 1rem 2rem;
+  align-items: center;
 }
-
-.menu-item {
-  flex: 1;
-  text-align: center;
+.menu-header {
+  flex-grow: 1;
   text-decoration: none;
   display: flex;
+  gap: 10px;
   align-items: center;
-  justify-content: center;
+  font-size: 2rem;
+}
+.menu-header img {
+  height: 50px;
+  width: 50px;
+}
+.menu-item {
+  margin: 0 1rem;
+  text-decoration: none;
```

Comment:
Nice work with the styling! Keep in mind though that we want to modularize our work and not exceed the boundaries of the targeted issue. I'll allow it this time, but for future reference, if you want to modify something else (like styling the navigation menu), please open a new issue so that you/other contributers can work on it